Makefile.in

```diff
@@ -142,7 +142,7 @@ basic_ldflags := $(debug_ldflags) $(cuda_ldflags) -L. -L@CHARM_PATH@/lib
 # ------- Standard Variables --------------------------------------------------
 # The global version of each flag must be at the end so that the user
 # can override our defaults here from the command line
-cpp_flags := $(includes) $(defines) @USER_CPPFLAGS@ $(CPPFLAGS)
+cpp_flags := $(includes) $(defines) @USER_CPPFLAGS@ @CPPFLAGS@
```

Comment:
That's weird. I distinctly remember moving these into the $USER_CPPFLAGS in the configure file. In any case, we also want to include the CPPFLAGS from the environment here in case a user does something like

    CPPFLAGS=foo ./configure ...
    CPPFLAGS=bar make ...

so that we capture the union of the two sets of CPPFLAGS. It's a simple addition:

    @USER_CPPFLAGS@ @CPPFLAGS@ $(CPPFLAGS)

Reply:
Just added this (and similar for LDFLAGS and LIBS).